modules/programs/mpv.nix

```
@@ -55,8 +55,13 @@ let
 
   mpvPackage = if cfg.scripts == [ ] then
     cfg.package
+  else if hasAttr "wrapMpv" pkgs then
+    pkgs.wrapMpv pkgs.mpv-unwrapped { scripts = cfg.scripts; }
   else
-    pkgs.wrapMpv pkgs.mpv-unwrapped { scripts = cfg.scripts; };
+    pkgs.mpv-unwrapped.wrapper {
+      mpv = pkgs.mpv-unwrapped;
+      scripts = cfg.scripts;
+    };
```

I believe this can be:
```suggestion
    pkgs.mpv.override { scripts = cfg.scripts; };
```
Besides that, LGTM. Thanks!

---

From what I could tell, `mpv-unwrapped.wrapper` replaces `wrapMpv` which seems to be necessary.

 I didn't really look into it that much, but if a simple override works, why isn't that the first thing we do here?

---

I'm not sure why, they yield the same derivation